test/libsolidity/semanticTests/structs/struct_delete_storage_nested_small.sol

```diff
@@ -33,4 +33,4 @@ contract C {
 // compileViaYul: true
 // ----
 // f() -> 0, 0, 0
-// gas irOptimized: 117101
+// gas irOptimized: 101084
```

Comment:
Interesting. That's 15%+ saving. Would be interesting to see what exactly changed here. Is the optimizer able to eliminate some storage writes?

Reply:
Ow! Nice catch. Yeah, I thought that the gas changes were a bit inconsistent but didn't investigate it further. Thanks a lot for that ;)

Reply:
For the record, I fixed the stale costs in #16348.